Makefile

```diff
@@ -170,6 +170,19 @@ release: static gen-device
 	./build/tinygo build-builtins -target=armv7em-none-eabi -o build/release/tinygo/pkg/armv7em-none-eabi/compiler-rt.a
 	tar -czf build/release.tar.gz -C build/release tinygo
 
+# Override this to match the version of software being released.
+RELEASE_VERSION ?= 1.0.0
+
+# Override this to match the arch for software being released.
+RELEASE_ARCH ?= amd64
+
+# Override this to match the tar with the software being released.
+RELEASE_TAR ?= ./build/release.tar.gz
+
+# Create debian DEB installer file. Requires fpm (https://github.com/jordansissel/fpm)
+deb:
+	fpm -s tar -t deb -n tinygo -v $(RELEASE_VERSION) -a $(RELEASE_ARCH) --prefix=/usr/local --deb-custom-control=./DEBIAN/control $(RELEASE_TAR)
```

Comment:
I wonder whether the prefix should be `/usr` instead of `/usr/local`?
This is a system-installed package so there is no system-installed tinygo it could conflict with (or it has been resolved by dpkg already). However, installing in `/usr/local` may conflict with someone that installs tinygo there.

Reply:
I selected that location since it is the same that Go itself uses if you install from the package manager `/usr/local/go`.